### ISSUE TITLE
New v1.1.2 release including cchardet dependency fix

### ISSUE THREAD

**Issue by FlorianNachtigall** [remix/partridge#82]
Any chance of publishing a new release incorporating the fix for the `cchardet` support (https://github.com/remix/partridge/issues/73)?

Would be amazing. Thanks in advance!

**Comment by dchimeno** [comment]
> Any chance of publishing a new release incorporating the fix for the `cchardet` support (#73)?
> 
> Would be amazing. Thanks in advance!

I've managed to install locally with:

```
python -m pip install 'partridge @ git+https://github.com/remix/partridge@master
```
but yes, a release could be great.

**Comment by FlorianNachtigall** [comment]
Thanks @dchimeno. Yes, this works well for development, but it unfortunately does not allow publishing to PyPI as they do not support non-package dependencies (e.g. see https://github.com/python-poetry/poetry/issues/2828).

**Comment by harto** [comment]
Fixed in #83
